src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,54 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private static final int ARRAY_SIZE = 10;
+    private K key;
+    private V value;
+    private final Object[] keys = new Object[ARRAY_SIZE];
+    private final Object[] values = new Object[ARRAY_SIZE];
+    private int exist = -1;
+    private int keysSize = -1;
```

Comment:
I think this will be better 
```suggestion
    private int size = 0;
```